CONTRIBUTING.adoc

```diff
@@ -132,15 +132,33 @@ To facilitate rapid development we've put together a Vagrantfile you can use to
 6.  Run a build in SSH:
 
         $ cd /data/src/github.com/openshift/origin
-        $ make build
+        $ make clean build
 
-7.  Start an OpenShift all-in-one server in SSH (includes everything you need to try OpenShift)
+7.  Now change into the directory with the OpenShift binaries, and start the OpenShift server:
 
-        $ sudo systemctl start openshift
+        $ cd _output/local/go/bin
+        $ sudo ./openshift start
+        $ sudo ./openshift start --public-master=localhost &> logs/openshift.log &
+
++
+NOTE: when using vagrant synced folder it is advised to use different directory than the one in synced folder. This can be achieved by passing `--volume-dir=/absolute/path` to `openshift start` command.
```

Comment:
not being  a heavy vagrant user, is it obvious which is the "synced" folder?

also at a minimum it should be "than the one in *the* synced folder"